client/src/pages/ViewEvent/ViewSpecialGroups.tsx

```diff
@@ -24,12 +24,17 @@ function processSpecialGroups(
       (sg) => sg.id === se.specialGroupId
     );
     if (specialGroupsFiltered.length === 0) {
-      console.log(`No special group found with matching id for special event with id ${se.specialGroupId}`);
+      console.log(
+        `No special group found with matching id for special event with id ${se.specialGroupId}`
+      );
```

Comment:
Do you want to use the logger here? It might fit more with this PR than the modal one.

Reply:
Good point, will do that